modules/metering-upgrade-verify.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-upgrading-metering.adoc
+
+[id="metering-upgrade-verify_{context}"]
+= Verify Metering Upgrade
+
+To verify your metering upgrade, check that all the required Pods have been created. Additionally, you can check that your `ReportDataSources` are importing new data.
+
+.Procedure
+
+.  Navigate to *Workloads* -> *Pods* in the metering namespace and verify that Pods are being created. This can take several minutes after upgrading the metering stack.
++
+You can run the same check using the `oc` CLI:
++
+----
+$ oc -n openshift-metering get pods
+----
++
+The output indicates that Pods are being created:
++
+----
+NAME                                  READY   STATUS              RESTARTS   AGE
+hive-metastore-0                      1/2     Running             0          52s
+hive-server-0                         2/3     Running             0          52s
+metering-operator-68dd64cfb6-pxh8v    2/2     Running             0          2m49s
+presto-coordinator-0                  2/2     Running             0          31s
+reporting-operator-56c6c878fb-2zbhp   0/2     ContainerCreating   0          4s
+----
+
+.  Continue to check your Pods until all containers in a Pod are reporting as `Ready`. This can take several minutes. Many Pods rely on other components to function before they themselves can be considered ready. Some Pods may restart if other Pods take too long to start, this is okay and can be expected during the Metering Operator upgrade.
```

Comment:
@lbarbeevargas `.` at the beginning of line 31 is a typo?

Reply:
Thanks, @pruan-rht. The `.` at the beginning of [several] lines is how we denote a new list item in Asciidoc syntax.